CHANGELOG.md

```diff
@@ -31,6 +32,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [1.7.0] - 2024-07-02
 
+- The global library path is now included in unit import resolution.
+
+## [1.7.0] - 2024-07-02
+``
```

Comment:
I think this was messed up in the merge conflict.

Reply:
Oops! Fixed that.